src/intentLogic/getLatestReviews.js

```diff
@@ -0,0 +1,86 @@
+const get = require('simple-get-promise').get;
+const asJson = require('simple-get-promise').asJson;
+
+const helpers = require('../helpers');
+const speech = require('../speech').speech;
+const sound = require('../speech').sound;
+const getMoreOffset = require('../helpers').getMoreOffset;
+
+module.exports = function(isNewIntentFlag) {
+    const isNewIntent = isNewIntentFlag !== false;
+
+    const attributes = this.event.session.attributes;
+    const slots = this.event.request.intent.slots;
+
+    attributes.lastIntent = 'GetLatestReviewsIntent';
+
+    if (isNewIntent) attributes.reviewType = slots.review_type.value;
+
+    if (attributes.reviewType) {
+        attributes.moreOffset = getMoreOffset(isNewIntent, attributes.moreOffset);
+
+        const filter = "page="+ ((attributes.moreOffset / helpers.pageSize) + 1)
+                     + "&page-size="+ helpers.pageSize
+                     + "&tag=tone/reviews,"+ getTagType(attributes.reviewType)
+                     + "&show-fields=standfirst,byline,headline&show-blocks=all";
+
+        const capiQuery = helpers.capiQuery('search', filter);
+        get(capiQuery)
+            .then(asJson)
+            .then(json => {
+                if (json.response.results && json.response.results.length > 1) {
+                    attributes.positionalContent = json.response.results.map(review => review.id);
+                    const preamble = getPreamble(isNewIntent, json.response.results.length, attributes.reviewType);
+                    const reviews = json.response.results.map(review => {
+                        return review.fields.headline + sound.transition;
+                    });
+                    const conclusion = getConclusion(json.response.results.length);
+
+                    this.emit(':ask', `${preamble} ${reviews} ${conclusion}`);
+                } else {
+                    this.emit(':tell', speech.reviews.notfound);
+                }
+            })
+            .catch(error => {
+                this.emit(':tell', speech.reviews.notfound);
+            })
+    } else {
+        this.emit(':ask', speech.reviews.clarifyType);
+    }
+}
+
+function getTagType(reviewType) {
+    switch (reviewType) {
+        case 'film':
+            return 'film/film';
+            break;
+        case 'restaurant':
+            return 'lifeandstyle/restaurants';
+            break;
+        case 'book':
+            return 'books/books';
+            break;
+        case 'music':
+            return 'music/music';
+            break;
+    }
+}
+
+function getPreamble(isNewIntent, reviewCount, reviewType) {
+    if (isNewIntent) {
+        return `Here are the latest ${reviewType} reviews.`;
+    } else {
+        if (reviewCount === 1) {
+            return `The next ${reviewType} review is`;
+        } else {
+            return `The next ${reviewType} reviews are`;
+        }
+    }
+}
+
+function getConclusion(reviewCount) {
```

Comment:
This is deliberate. I've been doing it this way since I saw this: http://stackoverflow.com/questions/22939130/when-should-i-use-arrow-functions-in-ecmascript-6
Not sure it really makes any difference.